### PR TITLE
Add helper for SSMing to an instance type

### PR DESCRIPTION
**Pull request by ghost**
### Which issue does this PR correspond to?
https://github.com/grapl-security/issue-tracker/issues/955

### What changes does this PR make to Grapl? Why?
This adds a helper script to make it easier to ssh into an instance via ssm.

Previously I was either logging into the AWS console and looking up an instance id manually or running one of the existing ssm scripts to get the instance id and then logging out.

### How were these changes tested?
Run ./bin/aws/ssm_ssh.sh "Nomad Agent" to log into an instance.

### CODE REVIEW

**Review by wimax-grapl — APPROVED** [review]
but besides the naming thing, looks good

File: bin/aws/lib/ssm_tools.sh [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
 
+_get_server_instance() {
+    # shellcheck disable=SC2140
+    SERVER_INSTANCE_ID=$(
+        aws ec2 describe-instances \
+            --filters Name=tag:Name,Values="${SERVER_TYPE}" Name="instance-state-name",Values="running" \
+            --query="Reservations[0].Instances[0].InstanceId" \
+            --output=text
+    )
+    echo "${SERVER_INSTANCE_ID}"
```
Comment (wimax-grapl) [inline bin/aws/lib/ssm_tools.sh:11]: we could just remove the echo and remove the subshell - i tihnk it'll output the same thing - but i'm also not opposed to the over-verbosity of this

File: bin/aws/lib/ssm_tools.sh [hunk 2]
```diff
@@ -45,3 +50,21 @@ EOF
         --document-name AWS-StartPortForwardingSession \
         --parameters "${SSM_PARAMETERS}"
 }
+
+ssm_ssh() {
```
[the remark this answers is not in the document]
Comment (twunderlich-grapl) [reply bin/aws/lib/ssm_tools.sh]: While I suspect that the SSM Manager is basically creating a ssh session internally, I do agree that its important to clarify. I've renamed the script and the function

File: bin/aws/lib/ssm_tools.sh (hunk 1 above)
Comment (twunderlich-grapl) [reply bin/aws/lib/ssm_tools.sh:11]: I'd prefer the verbosity of this for now so that its clear what's going on.

**Pull request merged**